helm/ffc-demo-payment-service-core/templates/deployment.yaml

```diff
@@ -51,12 +54,6 @@ spec:
           value: {{ quote .Values.container.scheduleQueueEndpoint }}
         - name: Messaging__ScheduleQueueUrl
```

Comment:
I removed the payment/schedule queue URL in the node services, as it is built up from the SQS endpoint and the queue name for an AWS endpoint.

I think there's a risk that an old setting for the queue URL could override the queue and name settings if left in, Leaving it in does give some flexibility, though I'm not sure we'd need to set it to anything different.

Reply:
I don't think there's a scenario where they'd need to be different.  It's just another dependency to worry about keeping in sync.  I'll remove and update the code.

Reply:
I've removed the dependency.